maintainers/maintainer-list.nix

```diff
@@ -762,6 +762,12 @@
     githubId = 153175;
     name = "Andrew Marshall";
   };
+  amanse = {
+    email = "amansetiarjp@gmail.com";
+    github = "amanse";
+    githubId = 13214574;
+    name = "Aman Setia";
+  };
```

Comment:
Please add yourself as maintainer in a separate commit.

Reply:
sorry about that, does it need to be a seperate PR too or in this is fine?

Reply:
same pr is fine.

Reply:
done